pkg/models/active_node.go

```diff
@@ -2,17 +2,18 @@ package models
 
 //ActiveNode model
 type ActiveNode struct {
-	Id        int64  `json:"id"`
-	NodeId    string `json:"node_id"`
-	Heartbeat int64  `json:"heartbeat"`
-	Sequence  int32  `json:"sequence"`
+	Id           int64  `json:"id"`
+	NodeId       string `json:"node_id"`
```

Comment:
May be nodeId is ok but just wanted to call out that is will usually be hostname_port (just in case we have multiple grafana instances running on a host on different ports)